portfolio/src/main/webapp/script.js

```diff
@@ -46,3 +46,29 @@ function addRandomQuote() {
       delay: (el, i) => 70*i
     })
 }
+
+/**
+ * Sends an email when someone fills out the contact me page. 
+ */
+function contactEmail() {
+    // Obtain name, email, and message from the contact page.
+    const formData = {
+        'name': $('input[name=name]').val(),
+        'email': $('input[name=email]').val(),
+        'message': $('textarea[name=message]').val()
+    };
+    console.log(formData);
```

Comment:
Remove console log statements. Though since the email sending below is not working yet, you can also consider removing the email sending logic below (and add some statement like console.log("sending email now!")), and implement the email sending logic in an upcoming review after it's working? This change is already pretty big without the email sending logic.

Reply:
I removed the console log statements and replaced the email sending logic with "console.log("Sending email now.")" for now. I will change this to make a POST request to the server after I get the server set up next week.

Reply:
great, thanks!